### PR TITLE
chore: adopt language-agnostic naming conventions for Script Editor

### PR DESCRIPTION
**Pull request by wiedld**
Pre-condition of #6282  

Before adding ui e2es for the SQL script editor, would like to have the proper naming convention used. Is no longer the flux editor, is now the script editor. (a.k.a. would rather not write SQL tests that keep referring to `flux--` everything.)

Only re-namings. No other code changes.
Made each renaming per PR commit (see commit text).

## Checklist
- [x] A PR description, regardless of the triviality of this change, that communicates the value of this PR
- [x] [Well-formatted conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) that provide context into the change
- [x] Documentation updated or issue created (provide link to issue/PR)
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] Feature flagged, if applicable

### CODE REVIEW

**Review by appletreeisyellow — APPROVED** [review]
LGTM!

File: src/shared/components/FilterList/InjectionOption.tsx [hunk 1]
```diff
@@ -43,10 +43,10 @@ const FluxInjectionOption: FC<Props> = ({
   setHoverdFunction,
 }) => {
   // TODO: move this to a flag once design wants to branch it out further
```
Comment (hoorayimhelping) [inline src/shared/components/FilterList/InjectionOption.tsx:45]: ✂️ this comment

Comment (hoorayimhelping) [inline src/shared/components/FilterList/InjectionOption.tsx:45]: ✂️ this comment - doesn't look like it's referring to anything meaningful anymore

**Review by hoorayimhelping — APPROVED** [review]

File: src/shared/components/FilterList/InjectionOption.tsx (hunk 1 above)
Comment (wiedld) [reply src/shared/components/FilterList/InjectionOption.tsx:45]: Nice catch. 
Will cut it in a follow up PR. (Want to merge this before any conflicts merge. 😅 )

**Pull request merged**